assets/MessagingService.kt

```diff
@@ -0,0 +1,183 @@
+package com.getalby.mobile
```

Comment:
can you please put any android or ios code specifically in an android or ios folder?